app/main.cpp

```diff
@@ -214,8 +215,28 @@ int main(int argc, char *argv[])
                                    QLatin1String("max-frames"), QLatin1String("64"));
     parser.addOption(maxFrames);
 
+    QCommandLineOption verbose(QLatin1String("verbose"),
+                               QCoreApplication::translate(
+                                "main", "Set the value to 'warning'- to display warnings on the console."
+                                " Set the value to 'debug' - to display debugging information on the console."
+                                " Set the value to 'all' - to display warnings and debugging information on the console."
+                                " By default, all warnings and debugging information are suppressed."),
```

Comment:
I have to say that I don't like this - the warnings are there and they are often quite useful to understand what's going on when users report bugs. If we hide everything by default, then it becomes much harder for them to report useful bug reports.

I think if at all perfparser should adopt categorized logging more, and then maybe change a few spammy warnings to debug messages. Then one could just change the qtlogging.ini to opt in/out from messages.

This patch here looks like a hammer to just silence everything, but will also hide stuff - I dislike that. Can we please move it out of this series and discuss it in a standalone review?

Reply:
Ok